src/test/java/uk/gov/ons/census/fwmt/tests/acceptance/steps/CensusSteps.java

```diff
@@ -201,4 +209,41 @@ public void aMessageReceivedFromRmThatFailsToSendToTmAfterAttempts(Integer attem
   public void theErrorIsLoggedViaSPLUNKStoredInA(String queueName) {
     assertEquals(1, queueUtils.getMessageCount(queueName));
   }
+
+  @Given("RM sends a cancel case Household job request with case ID {string}")
+  public void tmHasAnExistingJobWithCaseID(String caseId) throws URISyntaxException, InterruptedException {
```

Comment:
for this test how do we ensure that the job thats going to be cancelled exists ?